modules/tinymce/CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Toggling fullscreen mode with the `fullscreen` plugin now also fires the `ResizeEditor` event #TINY-8701
```

Comment:
One thing I noted on the Zendesk ticket for this that doesn't look like it was relayed back into this ticket:
> This one could be a little tricker as looking at the change it contains both a bug fix and a "Changed" entry which normally requires it to be in a minor release. That said the "Changed" entry is just that the ResizeEditor event is fired when toggling fullscreen which could be argued was a bug anyways so it would probably be fine to backport.